multilinear_extensions/src/expression.rs

```diff
@@ -1004,7 +1051,7 @@ pub mod fmt {
         if only_one_limb {
             data[0].to_string()
         } else {
-            format!("{name}[{}]", data.join(","))
+            format!("[{}]", data.join(","))
```

Comment:
we do not need original naming, due to plonky3 field already implement better formatter. This logic is only serve old goldilocks package